circuits/transfer/src/execute/output.rs

```diff
@@ -32,6 +32,53 @@ impl CircuitOutput {
         self.note.value_commitment()
     }
 
+    pub fn pad() -> Self {
+        // Picked constants to produce a valid circuit from padded outputs
+        const R: JubJubScalar = JubJubScalar::from_raw([
+            0x67b6e93522122c6a,
+            0x818d44c9fbbed587,
+            0xc0bf02e714ed0295,
+            0x4f08a96ec6fd1db,
+        ]);
+
+        const A: JubJubAffine = JubJubAffine::from_raw_unchecked(
+            BlsScalar([
+                0x288a6872498d2d2e,
+                0x81aa13bbaea3f726,
+                0xc136533b78f649e5,
+                0x67fdcd9e4fc9dd7c,
+            ]),
+            BlsScalar([
+                0xd9e97d2386dcd902,
+                0x55adb1d584a3e4d9,
+                0xd2c280278a413bde,
+                0x28035b2392dbffe9,
+            ]),
+        );
+
+        const B: JubJubAffine = JubJubAffine::from_raw_unchecked(
+            BlsScalar([
+                0xae590591f544bb26,
+                0x55dc6a663a7ed884,
+                0x1db063bf7552bbcb,
+                0x4e221c08210cbd7b,
+            ]),
+            BlsScalar([
+                0x11a9b8b6cbdf231e,
+                0xd8e0ae12bd095f00,
+                0xa7b341fd1b3b3f26,
+                0x68eb6d64f88a979f,
+            ]),
+        );
+
```

Comment:
You are right, we are forced to use these magic values because we lack `const` functions. Blocked by: https://github.com/dusk-network/dusk-pki/pull/64/files